webui/src/store/modules/okr.js

```diff
@@ -51,6 +53,7 @@ export default {
 
         OBJECTIVES_FAILED(state, payload) {
             state.error = payload;
+            state.invalidOneDriveForBusinessLicense = false;
```

Comment:
Not related to this PS's topic, but it seems we need to reset objectives here: state.objectives = []

Reply:
I was thinking about it and not sure we should, because often we have transient errors. In this case OKRs can blink etc. So probably it's better to not reset objectives array here.